ingestion/src/metadata/cli/docker.py

```diff
@@ -276,9 +275,13 @@ def run_docker(  # pylint: disable=too-many-branches
                 "Stopping docker compose for OpenMetadata and removing images, networks, volumes..."
             )
             logger.info("Do you want to Delete the docker mounted volumes from host")
-            user_response = click.prompt("Please enter [y/N]", type=str)
+            user_response = input("Please enter [y/N]\n")
             if user_response == "y":
-                shutil.rmtree(MAIN_DIR)
+                try:
+                    shutil.rmtree(MAIN_DIR)
+                except FileNotFoundError as err:
+                    logger.debug(traceback.format_exc())
+                    logger.error(err)
```

Comment:
If the dir may have already been deleted, should we skip showing an error that could confuse the user? IMHO we should avoid printing error messages directly.

Reply:
👍